src/main/scala/ai/privado/exporter/JSONExporter.scala

```diff
@@ -34,15 +34,14 @@ import ai.privado.model.exporter.{
   SourceModel,
   SourceProcessingModel
 }
-import ai.privado.model.{Constants, InternalTag, PolicyThreatType}
+import ai.privado.model.{CatLevelOne, Constants, DataFlowPathModel, InternalTag, PolicyThreatType}
```

Comment:
remove the unused import

Reply:
Removed here: https://github.com/Privado-Inc/privado-core/pull/370/commits/ac73a384505ebd99f64ef649231a9b12f8e0a46a